quadro/quadro_tests.cpp

```diff
@@ -5,7 +5,7 @@ bool is_equal(const double d1, const double d2) {
     if (isnan(d1) && isnan(d2)) {
         return 1;
     }
-    else if ( (isnan(d1) && !isnan(d2)) || (!isnan (d1) && isnan(d2)) ) {
+    else if (isnan(d1) && !isnan(d2) || !isnan (d1) && isnan(d2)) {
         return 0;
     }
     else if (d1 == d2) {
```

Comment:
you forgot to compare with some precision

Reply:
resolved